CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## v0.5.3
+
+### Added
+
+- Support specifying the path to a rust-lang/rust clone at runtime with `RUST_SRC_REPO`
+
+### Changed
+
+- Make `--with-cargo` the default to allow bisecting past changes in rustc options. Add `--without-cargo` flag to use the old behavior.
```

Comment:
I put these on a single line even though I think that might be a "no-no" because they were part of the same change.